src/app/pages/my-review/my-review.component.scss

```diff
@@ -0,0 +1,221 @@
+.mat-icon {
+    font-size:30px !important;
+    height: 34px !important;
+    width: 34px !important;
+  }
+
+.mat-raised-button{
+    height:50px !important;
+
+    font-size:20px !important;
+}
+
+table {
+    margin-left: 110px;
+    width: 90%;
+  }
+  .col{
+    display:flex;
+  }
+  .mat-column-feedback {
+    word-wrap: break-word !important;
+    white-space: unset !important;
+    flex: 0 0 28% !important;
+    width: 28% !important;
+    overflow-wrap: break-word;
+    word-wrap: break-word;
+  
+    word-break: break-word;
+  
+    -ms-hyphens: auto;
+    -moz-hyphens: auto;
+    -webkit-hyphens: auto;
+    hyphens: auto;
+  }
+
+mat-paginator {
+    margin-left: 110px;
+    width: 90%;
+  }
+
+html, body{
+  height: 100%;
+  width: 100%;
+  font-family: 'Poppins', sans-serif;
+  color: #222;
```

Comment:
Why did you change the font here? The project root css already has that

Reply:
Did know you included that, fixed.